makefile

```diff
@@ -31,11 +33,18 @@ run-sim2play: all
 	cd run; ./simulator --headless &
 	cd run; ./soccer -sim -y & ./soccer -sim -b
 
+debug: all
+	$(GDB) ./run/soccer
+
+debug-sim: all
+	-pkill -f './simulator --headless'
+	cd run; ./simulator --headless &
+	gdb ./run/soccer
```

Comment:
Turns out lldb is in the repos. Assuming my lldb is the same as mac's:
`lldb -- ./run/soccer -sim` works, --args dosent unfortunately. I'll mess with it and get it to work in every scenario.